pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TableConfig.java

```diff
@@ -438,4 +439,49 @@ public int getReplication() {
     }
     return replication;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
No need to override them

Reply:
Yes, this is not required if we use the ZNode version.